contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -0,0 +1,209 @@
+import collections
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+from contentcuration.constants import content_kinds
+
+from contentcuration import models as ccmodels
+from kolibri.content import models as kolibrimodels
+
+import logging as logmodule
+logging = logmodule.getLogger(__name__)
+
+
+class EarlyExit(BaseException):
+    def __init__(self, message, db_path):
+        self.message = message
+        self.db_path = db_path
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('channel_id', type=str)
+        parser.add_argument('license_id', type=str)
+
+    def handle(self, *args, **options):
+        license_id = options['license_id']
+        channel_id = options['channel_id']
+
+        license = ccmodels.License.objects.get(pk=license_id)
+        try:
+            channel = ccmodels.Channel.objects.get(pk=channel_id)
+            # increment the channel version
+            raise_if_nodes_are_all_unchanged(channel)
+            mark_all_nodes_as_changed(channel)
+            assign_license_to_contentcuration_nodes(channel, license)
+            create_kolibri_license_object(license)
+            increment_channel_version(channel)
+            prepare_export_database()
+            # TODO: increment channel version numbers when we mark nodes as changed as well
+            map_content_tags(channel)
+
+            map_channel_to_kolibri_channel(channel)
+            map_content_nodes(channel.main_tree, license)
+            # use SQLite backup API to put DB into archives folder.
+            # Then we can use the empty db name to have SQLite use a temporary DB (https://www.sqlite.org/inmemorydb.html)
+
+        except EarlyExit as e:
+            logging.warning("Exited early due to {message}.".format(
+                message=e.message))
+            self.stdout.write("You can find your database in {path}".format(
+                path=e.db_path))
+
+
+def create_kolibri_license_object(license):
+    return kolibrimodels.License.objects.get_or_create(
+        license_name=license.license_name
+    )
+
+
+def increment_channel_version(channel):
+    channel.version += 1
+    channel.save()
+
+
+def assign_license_to_contentcuration_nodes(channel, license):
+    channel.main_tree.get_family().update(license_id=license.pk)
+
+
+def map_content_tags(channel):
+    logging.debug("Creating the Kolibri content tags.")
+
+    cctags = ccmodels.ContentTag.objects.filter(
+        channel=channel).values("tag_name")
+    kolibrimodels.ContentTag.objects.bulk_create(
+        [kolibrimodels.ContentTag(**vals) for vals in cctags])
```

Comment:
`kolibri.content.models.ContentTag` still uses the default PK mechanism (`AutoIntegerField`). Should I add in that UUIDField PK then on both content curation and kolibri?